### DISCUSSION TITLE
Visualize falkordb graph

### DISCUSSION THREAD

**Discussion by georgfaust** [FalkorDB/FalkorDB#474] [Q&A]
Is there a way to visualize a graph like described here?

https://developer.redis.com/explore/redisinsight/redisgraph

With redisinsight2 I only get back text output, no visualization. (I also have to prepend all queries with`GRAPH QUERY` which is not super handy)

**Comment by gkorland** [comment]
Yes, you can still get graphic results as long as you return nodes and edges and not just scalars.
Try: `GRAPH.QUERY MotoGP "MATCH (a)-[b]->(c) return a,b,c"` 

![image](https://github.com/FalkorDB/FalkorDB/assets/753206/9a4582fe-35e0-4e48-aaf6-b8afca427740)

**Reply by georgfaust** [reply]
this is what I get

<img width="607" alt="image" src="https://github.com/FalkorDB/FalkorDB/assets/10209240/13879f81-eb97-4841-8821-9ced0ac0594f">

Version:

<img width="215" alt="image" src="https://github.com/FalkorDB/FalkorDB/assets/10209240/d88ae032-f8e3-4acf-82e3-14aff82cf508">
